configs/neox_arguments.md

```diff
@@ -1056,6 +1056,16 @@ Parallelism Arguments
 
 
 
+- **sequence_parallel**: bool
```

Comment:
Is this what we want the arg to be called? 

do we want there to be an `is_sequence_parallel` boolean flag that gets set, a la `is_pipe_parallel`? (I think it's not necessarily needed but happy to do so)

Reply:
I like this as-is. No need for an `is_sequence_parallel` unless later on we need to handle such models differently like in the pipeline case.